cmd/config/errors.go

```diff
@@ -0,0 +1,39 @@
+package config
+
+import "fmt"
+
+const errorReadingYamlFileText = "error trying to read the YAML file: %w"
+const invalidConfigurationObjectErrorText = "error invalid configuration object: %w"
+const updateRoleErrorText = "error trying to update role: %w"
+const creatingRoleErrorText = "error trying to create role: %w"
+const deletingRoleErrorText = "error trying to delete role: %w"
+const errorGettingRolesText = "error getting existing roles: %w"
+const errorParsingGetConfigResponseText = "error trying to parse response: %w"
+
+func newErrorReadingYamlFile(err error) error {
+	return fmt.Errorf(errorReadingYamlFileText, err)
```

Comment:
I know you shared some of this with me offline, but I'm a little confused to see all of these moved into a function. I'm not sure what its helping us improve nor does it seem to meet the accepted [Armory convention](https://armory.slab.com/posts/coding-standard-proposal-package-level-error-definitions-imdq2e4w).

I think we'd want 
`var ErrYamlFileRead = errors.New("error trying to read the YAML file")`
then used like so
`fmt.Errorf("%w: %v", ErrYamlFileRead, err)`
because 
`errors.Is(fmt.Errorf("%w: %q", ErrYamlFileRead, err), ErrYamlFileRead)`

I'm not sure we need methods for all of these; maybe just one method:
```
func newErrorWithContext(custom error, raised error) {
 return fmt.Errorf("%w: %v", custom, raised)`
}
```
because is saves some repetition on the format.